src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -330,6 +330,7 @@ protected virtual bool IsAttributeConditionallyOmitted(NamedTypeSymbol attribute
                         }
 
                         BoundExpression boundNamedArgument = BindNamedAttributeArgument(argument, attributeType, diagnostics);
+                        this.Compilation.VariableUsePass.Visit(boundNamedArgument);
```

Comment:
>this.Compilation.VariableUsePass.Visit(boundNamedArgument); [](start = 24, length = 59)

It looks like this code path cover only named attribute arguments (assignments to field/properties of the attribute). We should cover constructor arguments as well.